site/_data/docs/lighthouse/toc.yml

```diff
@@ -1,13 +1,16 @@
 - url: /docs/lighthouse/overview/
-- url: /docs/lighthouse/seo/
-  title: i18n.docs.seo.intro
-  description: i18n.docs.seo.overview
-- url: /docs/lighthouse/best-practices/
-  title: i18n.docs.best-practices.intro
-  description: i18n.docs.best-practices.overview
 - url: /docs/lighthouse/performance/
   title: i18n.docs.performance.intro
-  description: i18n.docs.performance.overview
+  description: i18n.docs.performance.intro_description
```

Comment:
also switched all these to `intro_description`, which seems like what was intended (shorter blurb for the TOC, slightly longer overview on the individual pages)

Reply:
Works for me!